config/templates/pages/default.xml

```diff
@@ -5,7 +5,7 @@
 
     <key>default</key>
 
-    <view>templates/pages/default</view>
+    <view>pages/default</view>
```

Comment:
should we go for singular `page` as in the symfony demo? https://github.com/symfony/demo/tree/master/templates /cc @chirimoya @danrot

Reply:
Doesn't it feel weird to have one time singular and one plural? Would stay with it the way it is.

Reply:
would also stay with the plural version

Reply:
ok for me